doc/source/users/macro_environment.rst

```diff
@@ -0,0 +1,6 @@
+.. _macro-environment:
+
+=================
+Macro Environment
+=================
+
```

Comment:
To be done in separate PR

Reply:
Maybe we can keep this file already here and I will create a new task for filling this file.